lib/user.rb

```diff
@@ -0,0 +1,21 @@
+class User
+  attr_reader :name, :joke
+
+  def initialize(name, joke)
+    @name = name
+    @joke = []
+  end
+
+  def user
+    @name = "Sal"
+  end
+
+  def jokes
```

Comment:
We really wanted `jokes` to be an attr_reader method, so I would have expected my class to initialize something like this:

```ruby
class User
  attr_reader :name, :jokes

  def initialize(name)
    @name = name
    @jokes = []
  end
end
```

This way, a user starts with no jokes and can `learn(joke)` later